guides/source/testing.md

```diff
@@ -775,7 +774,7 @@ All of these keyword arguments are optional.
 Example: Calling the `:show` action, passing an `id` of 12 as the `params` and setting `HTTP_REFERER` header:
 
 ```ruby
-get :show, params: { id: 12 }, headers: { "HTTP_REFERER" => "http://example.com/home" }
+get show_url, params: { id: 12 }, headers: { "HTTP_REFERER" => "http://example.com/home" }
```

Comment:
I should probably fix the `update_url` below it as well. That should also be `article_url` if I'm not mistaken.